### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by bradenbiz**

### CODE REVIEW

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
- You still have quite a few `@skyux` dependencies that are not required for your project and can be removed (e.g. `@skyux/omnibar-interop`).

File: projects/popovers/src/public-api.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export * from './modules/popover/types/popover-placement';
+export * from './modules/popover/types/popover-position';
+export * from './modules/popover/types/popover-trigger';
+
+// Components and directives must be exported to support Angular's "partial" Ivy compiler.
+// Obscure names are used to indicate types are not part of the public API.
+export { SkyDropdownItemComponent as λ1 } from './modules/dropdown/dropdown-item.component';
+export { SkyDropdownButtonComponent as λ2 } from './modules/dropdown/dropdown-button.component';
+export { SkyDropdownComponent as λ3 } from './modules/dropdown/dropdown.component';
+export { SkyDropdownComponent as λ4 } from './modules/dropdown/dropdown.component';
```
Comment (Blackbaud-SteveBrush) [inline projects/popovers/src/public-api.ts:21]: You got a duplicate export, here, which is causing the strange build error.

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

File: projects/popovers/src/public-api.ts (hunk 1 above)
Comment (bradenbiz) [reply projects/popovers/src/public-api.ts]: fixed in last commit

**Comment by bradenbiz** [comment]
removed unnecessary `@skyux` dependencies

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**